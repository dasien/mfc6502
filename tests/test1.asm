; This is a comment
   ;So is this.

*=$2000
VAL1 = #10
VAL2 = $20
 LDY #$DF
 STY $2304
BEGIN LDY #0
 LDY $23
 LDA #$34
 ROL A
 TAX
 LDA #$90
 ADC #$10
 ORA #1
 PHA
 LDA #0
 PLA
 SEC
 SED
 LDA VAL1
 LDA VAL2
 BNE BEGIN
