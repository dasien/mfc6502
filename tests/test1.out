A0 DF
8C 04 23
A0 0
A4 23
A9 34
2A 
AA 
A9 90
69 10
09 1
48 
A9 0
68 
38 
F8 
A9 10
A5 20
D0 E6
